content/applications/sales/sales/invoicing/invoicing_policy.rst

```diff
@@ -4,78 +4,82 @@ Invoice based on delivered or ordered quantities
 
-- The **Invoice what is ordered** rule is used as **default mode** in Odoo Sales, which means that
-  customers will be invoiced once the sales order is confirmed.
+- The *Invoice what is ordered* rule is used as the default mode in Odoo *Sales*, which means
+  customers are invoiced once the sales order is confirmed.
+- The *Invoice what is delivered* rule invoices customers once the delivery is done. This rule is
+  often used for businesses that sell materials, liquids, or food in large quantities. In these
+  cases, the ordered quantity may differ slightly from the delivered quantity, making it preferable
+  to invoice the quantity actually delivered.
 
-- The **Invoice what is delivered** rule will invoice customers once the delivery is done. This rule
-  concerns businesses that sell materials, liquids or food in large quantities. In these cases, the
-  quantity might diverge a little bit and it is, therefore, preferable to invoice the quantity
-  actually delivered.
+Being able to have different invoicing options provides more flexibility.
 
-Being able to have different invoicing options allow you more flexibility. Indeed, you need to know
-exactly how to invoice your customers for different situations.
+Invoicing policy features
+=========================
 
-Activate these features
-=======================
+To activate the necessary invoicing policy features, go to :menuselection:`Sales app -->
+Configuration --> Settings`, and under the :guilabel:`Invoicing` heading, select an
+:guilabel:`Invoicing Policy` rule: :guilabel:`Invoice what is ordered` or :guilabel:`Invoice what is
+delivered`.
 
-Go to :menuselection:`Sales --> Configuration --> Settings` and under **Invoicing policy** choose
-the rule you want to apply.
-
-.. image:: invoicing_policy/invoicing_policy_1.png
+.. image:: invoicing_policy/invoicing-policy-setting.png
    :align: center
-   :class: img-thumbnail
-   :alt: How to choose your invoicing policy on Odoo Sales?
+   :alt: Choosing an invoicing policy on Odoo Sales.
 
 .. important::
-   If you decide to choose the **Invoice what is delivered** rule, you will not be able to
-   activate the feature called **Automatic invoice**, which automatically generates invoices when
-   the online payment is confirmed.
+   If the :guilabel:`Invoice what is delivered` rule is chosen, it is **not** possible to activate
+   the :guilabel:`Automatic Invoice` feature, which automatically generates invoices when an online
+   payment is confirmed.
 
-Choose an invoicing policy on a product form
-============================================
+Invoicing policy on product form
+================================
 
-From any product page, under the **Sales tab**, you will find the invoicing policy, which can be
-manually changed.
+On any product page (:menuselection:`Sales app --> Products --> Products --> Choose any desired
+product`), the :guilabel:`Invoicing Policy` option is located under the :guilabel:`General
```

Comment:
Creative use of menuselection 😄  
Here's my suggestion to fit a more typical use case for this tag.

```suggestion
On any product page via the :menuselection:`Sales app --> Products --> Products dashboard, locate the :guilabel:`Invoicing Policy` option located under the :guilabel:`General
```